### PR TITLE
FTP-1379: Design System Tokens: Scales added.

### PR DESCRIPTION
**Pull request by subhasishnathAxelerant**
https://axelerant.atlassian.net/browse/FTP-1379

### CODE REVIEW

**Comment by subhasishnathAxelerant** [comment]
Lint is failing onpush and pull. Which is fixed in https://github.com/axelerant-trainings-frontend/grocery-plus/pull/30 by Tushar.
CC: @skippednote

File: tailwind.config.js [hunk 1]
```diff
@@ -15,21 +15,41 @@ module.exports = {
...
       },
       height: {
         xxs: '35px',
         xs: '48px',
+        52: '52px',
         sm: '54px',
+        56: '56px',
         lg: '116px',
         xl: '120px',
         '2xl': '1214px',
```
Comment (skippednote) [inline tailwind.config.js:55]: Group all these together under spacing
Replace names with numbers (sm, lg, xl, etc).

**Review by skippednote — CHANGES_REQUESTED** [review]

File: tailwind.config.js (hunk 1 above)
Comment (subhasishnathAxelerant) [reply tailwind.config.js:55]: Done.

**Comment by skippednote** [comment]
@subhasishnathAxelerant Please rebase

**Pull request merged**